packages/gafl-webapp-service/src/__tests__/server.spec.js

```diff
@@ -52,31 +53,37 @@ describe('The server', () => {
     expect(serverDecorateSpy).toHaveBeenCalledWith('toolkit', 'redirectWithLanguageCode', expect.any(Function))
   })
 
-  it('addLanguageCodeToUri is called with request but not new prices', async () => {
-    createServer(catboxOptions)
-    const serverDecorateSpy = jest.spyOn(server, 'decorate').mockImplementation(() => {})
+  it.each([[uris.NEW_PRICES.uri], [uris.RECURRING_TERMS_CONDITIONS.uri]])(
+    'addLanguageCodeToUri is called with request but not %s',
```

Comment:
I realise this is adapting an existing test but I am not sure I understand the point of it – if we are hitting `/buy` I'm not sure why it would ever be calling `addLanguageCodeToUri` with these URIs? It might make more sense to hit these URIs and then make sure we are not calling `addLanguageCodeToUri` with a redirect.